RyanYoung25/CS377
# SQL Injection Fix

**Significant Changes**
1. Updated files to fix SQL Injection
2. Broken code is commented out

**How to Test**
1. Check for Magic URL
2. Check for SQL Injection in **Delete Calendar**
